dockerfiles/Dockerfile.openshift

```diff
@@ -64,6 +58,16 @@ COPY --from=node-prod-deps /usr/src/app/node_modules node_modules
 # NOTE: project files likely to change between dev builds
 COPY . .
 
+# Generate git version information
+RUN bash -l -c ./scripts/git_version_info.sh
+
+# Clean up anything we don't need anymore
+RUN apt-get purge -y git curl libcurl4 libcurl3-gnutls && \
+    apt autoremove -y && \
+    apt-get clean -y && \
+# Clean up the apt files and remove the .git directory from the container
```

Comment:
Did you mean to remove `.git` here?

Reply:
I think I originally was thinking about it, but it only saves like 4MB and it's lost me work more than once so I'd given up on the idea and just stopped trying. ;) I think I can just remove the comment.